app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def create
+    @user = User.find_by(email: session_params[:email])
+
+    if @user&.authenticate(session_params[:password])
+      login!
+      render json: {
+        logged_in?: true,
+        user: @user
+      }
+    else
+      render json: {
+        status: 401,
+        errors: ['No user found']
+      }
+    end
+  end
+
+  def is_logged_in?
```

Comment:
I'm a small bit confused by the purpose of this function. Would you be able to explain it to me?

Reply:
This is something I thought I'd have more use for, but I was wrong, removed now